apps/condo/domains/property/schema/Property.js

```diff
@@ -146,6 +149,29 @@ const Property = new GQLListSchema('Property', {
                 return data.inwork.count
             },
         },
+
+        totalTicketsCount: {
+            schemaDoc: 'Counter for all tickets',
+            type: Virtual,
+            resolver: async (item, _, context) => {
+                const { data, errors } = await context.executeGraphQL({
+                    query: Ticket.GET_ALL_OBJS_WITH_COUNT_QUERY,
```

Comment:
Mat be it's better to add `GET_COUNT_QUERY` and use it?
We should use a cache for such fields it's very heavy to evaluate %(